### PR TITLE
Refactor cards for simplicity

### LINKED ISSUE #519: About page cards should run two columns on mobile
The current flow is single column at mobile sizes and it leaves too much whitespace on the right. A two column layout would be better.

Screenshot from 2021-12-22 05-16-58

### LINKED ISSUE #520: Text alignment of names in About page cards is screwy
Sometimes the name is right aligned, sometimes center aligned, sometimes right align. This should be standardized.
Screenshot from 2021-12-22 05-18-38
Screenshot from 2021-12-22 05-18-31

### PR DESCRIPTION
**Pull request by palewire**
This patch simplifies the About page's cards. Everything should look mostly the same. Here's what I did:

- Broke out a new SCSS component for the cards
- Restructured all the card metadata to use the same structure
- Wrote a new card include for use in all sections of the page
- Refactored each section to use the new include
- Smoothed out the CSS so it works for all different permutations
- Deleted unused CSS
- Downloaded 160x160 cuts of all photos that were external hyperlinks
- Moved all photos into a shared assets/bios folder
- Set an explicit height and width for photos

This patch fixes #520 and fixes #519.

### CODE REVIEW

**Review by choldgraf — COMMENTED** [review]
This looks good in general! One question - it seems like the ordering has changed for the names. For example, see the "special election cohort":

https://deploy-preview-588--jupyter-github-io.netlify.app/about

My assumption is that we should use alpha-numeric ordering, but it doesn't seem this is the ordering we're using right now

**Comment by palewire** [comment]
I fix those ones at the top. The others were being sorted alphabetically by a templatetag. I just did the sort manually in the data and then cut the template logic. So I think the lower ones are okay.

**Comment by palewire** [comment]
@choldgraf I think that's fixed and done.

**Comment by krassowski** [comment]
It looks like this one got conflicts now. Feel free to overwrite the changes to images if that's easier for you - I can quickly re-run optimization once this is merged.

**Comment by palewire** [comment]
@krassowski, I think I caught it up with master.

File: _includes/biocard.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <a href="https://github.com/{{ include.gh_handle }}" rel="noopener noreferrer">GitHub</a>
+    {% endif %}
+    {% if include.linkedin %}
+      <a href="{{ include.linkedin }}" rel="noopener noreferrer">LinkedIn</a>
+    {% endif %}
+    {% if include.twitter_handle %}
+      <a href="https://twitter.com/{{ include.twitter_handle }}" rel="noopener noreferrer">Twitter</a>
+    {% endif %}
+  </div>
+  </div>
```
Comment (krassowski) [inline _includes/biocard.html:28]: Tiny suggestion: let's indent on every div for visual cues (we have a few divs which miss closing tags highlighted by the checker, indentation helps prevent this issues a bit).

```suggestion
  <div class="biocard-copy">
    <div class="biocard-header">{{ include.name }}</div>
    {% if include.affiliation %}
      <div class="biocard-affiliation">{{ include.affiliation }}</div>
    {% endif %}
    <div class='biocard-badges'>
      {% if include.personal %}
        <a href="{{ include.personal }}" rel="noopener noreferrer">Web</a>
      {% endif %}
      {% if include.gh_handle %}
        <a href="https://github.com/{{ include.gh_handle }}" rel="noopener noreferrer">GitHub</a>
      {% endif %}
      {% if include.linkedin %}
        <a href="{{ include.linkedin }}" rel="noopener noreferrer">LinkedIn</a>
      {% endif %}
      {% if include.twitter_handle %}
        <a href="https://twitter.com/{{ include.twitter_handle }}" rel="noopener noreferrer">Twitter</a>
      {% endif %}
    </div>
  </div>
```

**Review by krassowski — APPROVED** [review]
Just tiny comment on formatting the HTML template.

**Pull request merged**